test/sqllogictest/alter.slt

```diff
@@ -24,10 +24,10 @@ ALTER SYSTEM SET enable_rbac_checks TO false;
 ----
 COMPLETE 0
 
-query error system schema 'mz_catalog' cannot be modified
+query error must be owner of TABLE mz_catalog.mz_tables
 ALTER TABLE mz_tables RENAME TO foo;
```

Comment:
If we don't already have one, can you add a test to make sure renaming `mz_tables` also fails when running with `mz_system`?

Reply:
Done